scripts/skills/racial/rf_zombie_racial.nut

```diff
@@ -0,0 +1,99 @@
+this.rf_zombie_racial <- ::inherit("scripts/skills/skill", {
+	m = {
+		FatigueEffectMult = 0.0,
+		FatigueDealtPerHitMultModifier = 1.0
+	},
+	function create()
+	{
+		this.m.ID = "racial.rf_zombie";
+		this.m.Name = "Wiederganger";
+		this.m.Description = "This character is a wiederganger.";
+		this.m.Icon = "ui/orientation/zombie_01_orientation.png";
+		this.m.Type = ::Const.SkillType.Racial | ::Const.SkillType.StatusEffect;
+		this.m.Order = ::Const.SkillOrder.Last;
+	}
+
+	function getTooltip()
+	{
+		local ret = this.skill.getTooltip();
+
+		if (this.m.FatigueEffectMult == 0.0)
+		{
+			ret.push({
+				id = 10,
+				type = "text",
+				icon = "ui/icons/fatigue.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Builds no [Fatigue|Concept.Fatigue]")
+			});
+		}
+		else if (this.m.FatigueEffectMult != 1.0)
+		{
+			ret.push({
+				id = 10,
+				type = "text",
+				icon = "ui/icons/fatigue.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Build up " + ::MSU.Text.colorizeMult(this.m.FatigueEffectMult, {InvertColor = true}) + " less [Fatigue|Concept.Fatigue]")
+			});
+		}
+
+		local bonusFatiguePerHit = ::Const.Combat.FatigueReceivedPerHit * this.m.FatigueDealtPerHitMultModifier;
+		if (bonusFatiguePerHit != 0.0)
+		{
+			ret.push({
+				id = 11,
+				type = "text",
+				icon = "ui/icons/fatigue.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Inflicts " + ::MSU.Text.color(::Const.UI.Color.DamageValue, bonusFatiguePerHit) + " extra [Fatigue|Concept.Fatigue] on a hit")
+			});
+		}
+
+		ret.extend([
+			{
+				id = 20,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = "Not affected by nighttime penalties"
+			},
+			{
+				id = 21,
+				type = "text",
+				icon = "ui/icons/special.png",
+				text = ::Reforged.Mod.Tooltips.parseString("Not affected by, and cannot receive, [temporary injuries|Concept.InjuryTemporary]")
```

Comment:
I just realized we need to remove the `Not affected by` part here because in Reforged because of the `rf_undead_injury_receiver_effect`, zombies and skeletons **are** affected by temporary injuries present on them. They just can't receive them (unless the attacker has Crippling Strikes).